src/components/Root/Root.tsx

```diff
@@ -8,27 +8,24 @@ import { IS_DEMO_MODE, IS_DEVELOPMENT_BUILD, IS_PREVIEW_MODE } from '#src/utils/
 import DemoConfigDialog from '#components/DemoConfigDialog/DemoConfigDialog';
 import LoadingOverlay from '#components/LoadingOverlay/LoadingOverlay';
 import DevConfigSelector from '#components/DevConfigSelector/DevConfigSelector';
-import { useConfigSource } from '#src/utils/configOverride';
-import { loadAndValidateConfig } from '#src/utils/configLoad';
-import { initSettings } from '#src/stores/SettingsController';
 import AppRoutes from '#src/containers/AppRoutes/AppRoutes';
 import registerCustomScreens from '#src/screenMapping';
+import { initApp } from '#src/modules/init';
+import { useConfigSource } from '#src/utils/configOverride';
+import { useFeaturesStore } from '#src/stores/FeaturesStore';
 
 // This is moved to a separate, parallel component to reduce rerenders
 const RootLoader = ({ setAppIsReady }: { setAppIsReady: React.Dispatch<React.SetStateAction<boolean>> }) => {
   const { t } = useTranslation('error');
-  const settingsQuery = useQuery('settings-init', initSettings, {
-    enabled: true,
-    retry: 1,
-    refetchInterval: false,
-  });
 
-  const configSource = useConfigSource(settingsQuery?.data);
+  const configSource = useConfigSource();
 
-  const configQuery = useQuery('config-init-' + configSource, async () => await loadAndValidateConfig(configSource), {
-    enabled: settingsQuery.isSuccess,
+  const configQuery = useQuery('config-init-' + configSource, async () => initApp(configSource), {
+    enabled: !!configSource,
     retry: configSource ? 1 : 0,
     refetchInterval: false,
+    cacheTime: 1000 * 60 * 60 * 6,
```

Comment:
Does this change how quickly an app config change will get propagated without doing a page refresh? In some cases it's fine to assume the user will reload, but there may be cases where they won't. Maybe 20 minutes or an hour is a more realistic choice than 6 hours? It's also best practice I believe the make cache time slightly larger than stale time.

Reply:
We set it to 8 hours for series so I was focusing on approximately same value for the app config. We can define the common cache / stale interval for endpoints and set it to 20-30 minutes. What do you think?

Reply:
I wouldn't override it for all cases, but agreed that using something similar for series and this probably makes sense. Maybe you can just use a const like `CONTENT_CACHE_TIME` and `CONTENT_STALE_TIME`?